BootloaderCorePkg/Tools/GenCapsuleFirmware.py

```diff
@@ -481,7 +481,7 @@ def ValidateRegistryFormatGuid(Argument):
     parser = argparse.ArgumentParser()
 
     parser.add_argument('-p',  '--payload', nargs=2, action='append', type=str, required=True, help='Specify payload information including GUID, FileName')
-    parser.add_argument('-k',  '--priv_key', dest='PrivKey', type=str, required=True, help='Private RSA 2048 key in PEM format to sign image')
+    parser.add_argument('-k',  '--priv_key', dest='PrivKey', type=str, required=True, help='Key Id or Private RSA 2048 key in PEM format to sign image')

```

Comment:
Does this has to be 2048 bit key ?

Reply:
It can be RSA2K/3K keys. Updated help section.